src/tests/suite/daos_test_common.c

```diff
@@ -942,5 +958,38 @@ get_daos_prop_with_owner_acl_perms(uint64_t perms, uint32_t type)
 	prop->dpp_entries[0].dpe_type = type;
 	prop->dpp_entries[0].dpe_val_ptr = acl;
 
+	/* No need to free ACL - belongs to prop now */
+	return prop;
+}
+
+daos_prop_t *
+get_daos_prop_with_user_acl_perms(uint64_t perms,
+				  uint32_t owner_prop_type,
```

Comment:
owner_prop_type and acl_prop_type arguments are not used. Should they just be removed? Is the idea to somehow parameterize what would  typically(always?) be DAOS_PROP_CO_OWNER and DAOS_PROP_CO_ACL?

Reply:
I had an idea that this could be reused by pool tests (in which case the arguments would have to be the DAOS_PROP_PO_* variants), but in practice it's not used in the pool tests right now. I'll remove them.